react/src/components/Chat/ChatCard.tsx

```diff
@@ -240,9 +240,13 @@ const ChatCard: React.FC<ChatCardProps> = ({
     },
   });
 
+  const isStreaming = useMemo(() => {
+    return status === 'streaming' || status === 'submitted';
+  }, [status]);
+
```

Comment:
It is better not to use `useMemo` here because the calculation is so simple.
```suggestion
  const isStreaming = status === 'streaming' || status === 'submitted';
```